Build/Build.cs

```diff
@@ -92,7 +92,9 @@ class Build : NukeBuild
                     "Branch spec {branchspec} is a pull request. Adding build number {buildnumber}",
                     BranchSpec, BuildNumber);
 
-                SemVer = string.Join('.', GitVersion.SemVer.Split('.').Take(3).Union([BuildNumber]));
+                var prNumber = BranchSpec.Split("/").First(token => token.All(c => char.IsDigit(c)));
```

Comment:
My thoughts: we cannot rely on the number always on index 2 after split, but we can rely on the PR number is always "a number"

Reply:
Not needed...